CMakeLists.txt

```diff
@@ -170,6 +173,26 @@ else()
   target_link_libraries(rccl PUBLIC hip::hip_hcc ${hcc_LIBRARIES} numa)
 endif()
 
+## Set the VERSION and SOVERSION values
```

Comment:
from rocBLAS: rocm_set_soversion( rocblas ${rocblas_SOVERSION} )

Something like that will do this section more efficiently I believe.

Reply:
https://github.com/ROCmSoftwarePlatform/rocBLAS/search?q=rocblas_SOVERSION&unscoped_q=rocblas_SOVERSION

Reply:
Done.